rkt/run.go

```diff
@@ -92,6 +94,11 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if len(flagPodManifest) > 0 && (len(flagVolumes) > 0 || len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || rktApps.Count() > 0 || flagLocal) {
```

Comment:
The thought I have is since `image id` is required in pod manifest, then user must fetch and know the image id before `run --pod-manifest`, it cannot just throw the image name and let rkt fetch and run the pod manifest. This might not be a good UX IMO even though we assume the pod manifest is generated by some programs.